queries/python/highlights.scm

```diff
@@ -67,11 +67,11 @@
     (identifier) @type)) ; type subscript: Tuple[int]
 
 ((call
-  function: (identifier) @isinstance
+  function: (identifier) @function.builtin
   arguments: (argument_list
     (_)
     (identifier) @type))
- (#eq? @isinstance "isinstance"))
+ (#eq? @function.builtin "isinstance"))
```

Comment:
This is already captured here

https://github.com/nvim-treesitter/nvim-treesitter/blob/9c456edb3ad24be92a54b42545df8f0d0ee4c71b/queries/python/highlights.scm#L54-L57

Looks like the capture here was meant to be an auxiliary group. We can name it like `@_isinstance` or `@_function`.